editor/src/components/text-editor/text-editor.spec.browser2.tsx

```diff
@@ -781,8 +806,12 @@ async function pressShortcut(editor: EditorRenderResult, mod: Modifiers, key: st
   await editor.getDispatchFollowUpActionsFinished()
 }
 
-async function testModifier(mod: Modifiers, key: string) {
-  const editor = await renderTestEditorWithCode(projectWithoutText, 'await-first-dom-report')
+async function testModifier(
+  mod: Modifiers,
+  key: string,
+  startingProject: string = projectWithoutText,
+) {
+  const editor = await renderTestEditorWithCode(startingProject, 'await-first-dom-report')
 
   await prepareTestModifierEditor(editor)
   await pressShortcut(editor, mod, key)
```

Comment:
would it be worth it o use `expectSingleUndoStep` here?

Reply:
absolutely!!

Reply:
I'm happy to do it - in a separate PR after I fixed text editor undo